stable/jenkins/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: jenkins
 home: https://jenkins.io/
-version: 0.16.16
+version: 0.16.17
```

Comment:
Bump this one more step so we can merge https://github.com/helm/charts/pull/6987 then this one